src/app/shared/help-support/help-support.ts

```diff
@@ -0,0 +1,16 @@
+import {Component, NgModule} from '@angular/core';
+
+@Component({
+  selector: 'app-help-support',
+  templateUrl: './help-support.html',
+  styleUrls: ['./help-support.scss']
+})
+export class HelpSupport {
```

Comment:
We should probably just call this `Support` everywhere instead of both `HelpSupport`.